help_to_heat/frontdoor/views.py

```diff
@@ -557,20 +572,32 @@ def get_context(self, request, session_id, *args, **kwargs):
             {
                 "question": schemas.summary_map[question],
                 "answer": self.get_answer(session_data, question),
-                "change_url": reverse("frontdoor:change-page", kwargs=dict(session_id=session_id, page_name=page_name)),
+                "change_url": self.get_change_url(session_id, question, page_name),
             }
             for page_name, questions in schemas.household_pages.items()
             for question in questions
-            if question in session_data
-            if question in schemas.summary_map
+            if self.show_question(session_data, question)
         )
         return {"summary_lines": summary_lines}
 
+    def show_question(self, session_data, question):
+        show_property_type_lines = self.get_answer(session_data, "property_type") != "Park home"
+        question_answered = question in session_data and question in schemas.summary_map
+        if question == "property_type" or question == "property_subtype":
+            return show_property_type_lines and question_answered
+        else:
+            return question_answered
+
     def get_answer(self, session_data, question):
         answer = session_data.get(question)
         answers_map = schemas.check_your_answers_options_map.get(question)
         return answers_map[answer] if answers_map else answer
 
+    def get_change_url(self, session_id, question, page_name):
+        if question == "park_home":
+            return reverse("frontdoor:page", kwargs=dict(session_id=session_id, page_name=page_name))
```

Comment:
Need to go through the journey again if one attempts to change park home answer from summary page (confirmed with Ladun